src/cursor.rs

```diff
@@ -100,20 +101,31 @@ impl Cursor {
     /// # Return value
     ///
     /// Returns the newly created Cursor on success, or an Error on failure.
-    pub fn command_cursor(client: Client, db: &str,
-                          doc: bson::Document, cmd_type: CommandType,
-                          read_pref: ReadPreference) -> Result<Cursor> {
-        Cursor::query(client.clone(), format!("{}.$cmd", db), 1, OpQueryFlags::no_flags(), 0, 0,
-                      doc, None, cmd_type, true, read_pref)
+    pub fn command_cursor(client: Client,
+                          db: &str,
+                          doc: bson::Document,
+                          cmd_type: CommandType,
+                          read_pref: ReadPreference)
+                          -> Result<Cursor> {
+        Cursor::query(client.clone(),
+                      format!("{}.$cmd", db),
+                      1,
+                      OpQueryFlags::no_flags(),
+                      0,
+                      0,
+                      doc,
+                      None,
+                      cmd_type,
+                      true,
+                      read_pref)
```

Comment:
not gonna tackle how ridiculously long our `query` fn is, and how rust doesn't have named args :(